packages/asc-ui/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react'
+import { ChevronLeft, ChevronRight } from '@datapunt/asc-assets'
+import PaginationStyle, {
+  Props,
+  PageNumber,
+  PreviousButton,
+  NextButton,
+} from './PaginationStyle'
+
+const Pagination: React.FC<Props & React.HTMLAttributes<HTMLElement>> = ({
+  page = 1,
+  collectionSize,
+  pageSize = 10,
+  onPageChange,
+  ...otherProps
+}) => {
+  const [currentPage, setCurrentPage] = React.useState<number>(page)
+  const totalPages = Math.ceil(collectionSize / pageSize)
+  const pageCluster = [currentPage]
+  if (currentPage > 2) {
+    pageCluster.unshift(currentPage - 1)
+  }
+  if (totalPages <= 5 || currentPage < totalPages - 1) {
+    pageCluster.push(currentPage + 1)
+  }
+
+  React.useEffect(() => {
+    if (onPageChange !== undefined) onPageChange(currentPage)
+  }, [currentPage, onPageChange])
+
+  const handlePrevious = () => {
+    setCurrentPage(currentPage - 1)
+  }
+
+  const handleNext = () => {
+    setCurrentPage(currentPage + 1)
+  }
+
+  const renderButton = (pageNumber: number) => {
+    const isCurrent = pageNumber === currentPage
+    return (
+      <PageNumber
+        aria-label={
+          isCurrent ? `Pagina ${pageNumber}` : `Ga naar pagina ${pageNumber}`
+        }
+        aria-current={isCurrent}
+        data-testid={`btn-page-${pageNumber}`}
+        onClick={(evt) => {
+          // Remove focus from button after click
+          evt.currentTarget.blur()
+          setCurrentPage(pageNumber)
+        }}
+        isCurrent={isCurrent}
+        tabIndex={isCurrent ? -1 : 0}
+      >
+        {pageNumber}
+      </PageNumber>
+    )
+  }
+
+  const renderUnphased = () => {
+    return (
+      <>
+        {Array.from(Array(totalPages).keys()).map((num) => (
+          <li key={`pag-${num}`}>{renderButton(num + 1)}</li>
+        ))}
+      </>
+    )
+  }
+
+  const renderPhased = () => {
+    return (
+      <>
+        {currentPage > 1 && <li key="pag-1">{renderButton(1)}</li>}
+        {currentPage > 3 && (
+          <li key="pag-spacer-1" className="spacer">
+            ...
```

Comment:
I think we should use the `&mldr;` ellipsis punctuation character here for semantic purposes

Reply:
Do you have a suggestion on how to work around this?
![image](https://user-images.githubusercontent.com/1095870/122346219-034ef300-cf49-11eb-9007-9aabb661d5d7.png)